test/BUILD.gn

```diff
@@ -91,6 +91,7 @@ test("brave_unit_tests") {
   ]
 
   sources = [
+    "//brave/app/command_utils_unittest.cc",
```

Comment:
AFAIK, we don't recommend to add indivisual test files directly to this grand target.
Insstead, we could have `//brave/app:unit_tests` target` and `//brave/app/browser_tests`.